doc/doxygen/src/ForManagers.dox

```diff
@@ -135,7 +135,7 @@
  *
  * - Implement the methods grouped under the "Publishing" section, and
  *   the remaining methods in the "Entity Information" section of
- *   @ref openassetio.managerApi.ManagerInterface.ManagerInterface
+ *   @fqref{managerApi.ManagerInterface}.ManagerInterface
```

Comment:
[fix] This links to the `ManagerInterface` constructor, probably not what was intended.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1058/commits/948efe5518f4da207ec3438f17531c456b524965